### ISSUE TITLE
Non-default keyboard shortcuts are not shown properly in the keyboard quick reference

### ISSUE THREAD

**Issue by OfekShilon** [rstudio/rstudio#6951]
### System details
    RStudio Edition : Server 
    RStudio Version : 1.2.5042
    OS Version      :  Ubuntu 14.04.5 LTS
    R Version       : 3.4.0 (2015-04-16)

### Steps to reproduce the problem
1. Customize any keyboard shortcut, via Tools/ Modify keyboard shortcuts.
2. Display 'Keyboard shortcut help' via the tools menu or Alt+Shift+K

### Describe the problem in detail
The displayed shortcut is the original (non modified one).

### Describe the behavior you expected
The displayed shortcut should be the modified one.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Let's track this via the existing issue #5256. I have updated the title to mention this.
